src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.cpp

```diff
@@ -1666,6 +1666,44 @@ void ReconstructionEngine_sequentialSfM::getTracksToTriangulate(const std::set<I
   }
 }
 
+namespace {
+
+struct ObservationParams
```

Comment:
I would call it more `ObservationData` rather than Params as it is more the input data rather than parameters that control the algorithm